src/ILLink.Tasks/LinkTask.cs

```diff
@@ -331,7 +331,7 @@ protected override string GenerateResponseFileCommands ()
 			if (clearInitLocals) {
 				args.AppendLine ("--enable-opt clearinitlocals");
 				if (ClearInitLocalsAssemblies?.Length > 0) {
-					args.AppendFormat ($"--custom-data ClearInitLocalsAssemblies={ClearInitLocalsAssemblies}");
+					args.AppendFormat ($"--custom-data ClearInitLocalsAssemblies={ClearInitLocalsAssemblies} ");
```

Comment:
maybe use AppendLine instead?

Reply:
I hate the newlines in the log they are a pain to deal with (merging to unblock darc update)